homeassistant/components/tradfri/base_class.py

```diff
@@ -34,22 +40,26 @@ class TradfriBaseClass(Entity):
 
     _attr_should_poll = False
 
-    def __init__(self, device, api, gateway_id):
+    def __init__(
+        self, device: Command, api: Callable[[str], Any], gateway_id: str
+    ) -> None:
         """Initialize a device."""
         self._api = handle_error(api)
-        self._device = None
-        self._device_control = None
-        self._device_data = None
+        self._device: Command | None = None
+        self._device_control: SocketControl = None
```

Comment:
Same discussion as in https://github.com/home-assistant/core/pull/56002#pullrequestreview-754757198: Should it maybe be `SocketControl | None`?

Reply:
seems the latest change never made it to GitHub. It is with None.